core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -359,6 +359,22 @@ public static ByteString fromHex(String hex) {
     }
   }
 
+  /** SQL HEX(varchar) function. */
```

Comment:
nit: could you please put the new methods after the existing `toHex` method definition? I'd like to keep "related" methods (fromHex/toHex) right next to each other (similarly to the new ones hex/unhex).

Reply:
Done